### PR TITLE
Jenkins: Add build scripts

### PR DESCRIPTION
**Pull request by NicolasT**

### CODE REVIEW

**Comment by scality** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:7443//job/srb-basic/1/
Test failed.

File: jenkins/srb-basic/10-install-deps.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        sudo yum install -y kernel-devel-$(uname -r)
+        sudo yum install -y make gcc
+}
+
+function detect_platform() {
+        if test -e /etc/redhat-release; then
+                echo redhat
+                return
+        fi
+        if grep ^DISTRIB_ID=Ubuntu$ /etc/lsb-release > /dev/null 2>&1; then
```
Comment (JordanP) [inline jenkins/srb-basic/10-install-deps.sh]: lsb_release -s -i maybe ?

**Comment by scality** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:7443//job/srb-basic/2/
Test failed.

**Comment by scality** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:7443//job/srb-basic/3/
Test failed.

**Comment by scality** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:7443//job/srb-basic/4/
Test failed.

**Comment by NicolasT** [comment]
Anyone watching this: it's normal tests are failing, these scripts are developed 'through CI'.

**Comment by scality** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:7443//job/srb-basic/5/
Test failed.

**Comment by scality** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:7443//job/srb-basic/6/
Test failed.

**Comment by scality** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:7443//job/srb-basic/8/
Test failed.

**Comment by NicolasT** [comment]
Retest this please.

**Comment by scality** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:7443//job/srb-basic/9/
Test passed.

**Comment by NicolasT** [comment]
Jenkins, retest this please.

**Comment by scality** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:7443//job/srb-basic/11/
Test passed.

**Pull request merged**